collections/Tanner/MS_Tanner_346.xml

```diff
@@ -37,57 +40,203 @@
                      <idno type="SCN">10173</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Anthology of courtly poetry in English; England, s. xv. med</head>
                <msContents>
+                  <textLang mainLang="en">Middle English</textLang>
+                  <msItem>
+                     <locus>(fol. i r – ii v)</locus>
+                     <note>Modern endleaf, blank.</note>
+                     </msItem>
+                  <msItem><locus>(fol. iii r)</locus>
+                     <note>Modern endleaf, table of contents, headed ‘Some of Chaucer’s Works’, 17th century.</note>
+                     </msItem>
+                  <msItem>
+                     <locus>(fol. iii v – iv v)</locus>
+                     <note>Modern endleaf, blank.</note>
+                  </msItem>
                   <msItem n="1" xml:id="MS_Tanner_346-item1">
-                     <author key="person_100185203">Geoffrey Chaucer</author>
-                     <title key="work_1665">poems</title>
-                     <textLang mainLang="en">English</textLang>
+                     <locus>(fols. 1r-40v)</locus>
+                     <author key="person_100185203"><supplied>Geoffrey Chaucer</supplied></author>
```

Comment:
The use of `<supplied>` is a good idea, but of course few if any of our other records do this – do you think we should keep it? This also begs the question of the extent to which we should be further separating the description of manuscripts and works.